src/design/themes.stories.mdx

```diff
@@ -23,15 +13,17 @@ Themes may be applied to a containing element to change that content's appearanc
 The default theme, applied to the document `:root` by default.
 
 <Canvas>
-  <Story name="Light">{demoStory.bind({})}</Story>
+  <Story name="Light" args={{ theme: 'light' }}>
+    {demoStory.bind({})}
+  </Story>
 </Canvas>
 
 ## Dark
 
 Optimized for page intros and headings.
 
 <Canvas>
-  <Story name="Dark" args={{ theme: 'dark' }}>
+  <Story name="Dark" parameters={{ theme: 't-dark' }} args={{ theme: 'dark' }}>
```

Comment:
Should this be `dark` or `t-dark`? You used `t-dark` in `button.stories.mdx`.

Reply:
It's kind of confusing, but I'm using `args` to tell the demo which theme is applied, and `parameters` to tell the decorator which theme to use on the container.

Storybook's weird.